src/shared/common/forms/__tests__/__snapshots__/date.spec.js.snap

```diff
@@ -656,4 +656,4 @@ exports[`Form date component should render with value when passed 1`] = `
     </div>
   </fieldset>
 </div>
-`;
+`;
```

Comment:
Always leave the last `\n` on final lines. You've accidentally removed it here -- remove this file from the commit.

Reply:
Good spot added back in